packages/plugins/users-permissions/admin/src/pages/AdvancedSettings/index.js

```diff
@@ -169,12 +169,12 @@ const AdvancedSettingsPage = () => {
                   paddingRight={7}
                 >
                   <Stack size={4}>
-                    <H3>
+                    <Typography variant="delta" as="h2">
```

Comment:
Shouldn't it be a h3 ?

Reply:
I'm not sure, the view only has an h1 (Advanced Settings title) so I thought Settings should be h2 - do you think h3 is better?

<img width="615" alt="Screenshot 2021-11-15 at 18 29 13" src="https://user-images.githubusercontent.com/71838159/141827136-8306a944-8599-43f7-b65f-bc82c273a6fd.png">

Reply:
Yes you are right ! Thanks 😄